projects/storefrontlib/src/shared/components/form/form-errors/form-errors.component.ts

```diff
@@ -17,8 +19,13 @@ import { map, startWith } from 'rxjs/operators';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class FormErrorsComponent {
+  _parent: FormGroupDirective;
   _control: FormControl;
-  errors$: Observable<string[]>;
+  error$: Observable<string>;
+
+  constructor(@Optional() @Host() parent: FormGroupDirective) {
+    this._parent = parent;
```

Comment:
Why not make the argument protected? also would rename it to express the value better, i.e. 

```ts
@Optional() @Host() protected formGroup: FormGroupDirective
```